### PR TITLE
Flink Delta Sink - Update Readme

### PR DESCRIPTION
**Pull request by pkubit-g**

### CODE REVIEW

**Comment by scottsand-db** [comment]
@pawel-kubit-cs  this seems stuck in `Checking for ability to merge automatically…` state. can you push an empty commit?

File: flink-connector/README.md [hunk 1]
```diff
@@ -14,15 +14,18 @@ utilizing [Delta Standalone JVM library](https://github.com/delta-io/connectors#
...
 - depending on the version of the connector you can use it with following Apache Flink versions:
   
   | connector's version  | Flink's version |
   | :---: | :---: |
   |    0.2.1-SNAPSHOT    |    >= 1.12.0    |
 
+#### Known limitations:
+- currently only DeltaSink is supported which means that this connector is only ably to write data to a Delta table and is not able to read from it,
+- current version supports only Flink Datastream API. Support for Flink Table API / SQL along with Flink Catalog's implementation for storing Delta table's metadata in an external metastore are planned to be added in the next releases,
+- current version provides Delta's transactional gurantess only for tables stored on HDFS and Microsoft Azure Storage
```
Comment (allisonport-db) [inline flink-connector/README.md:27]:
```suggestion
- current version provides Delta's transactional guarantees only for tables stored on HDFS and Microsoft Azure Storage 
```

**Comment by pkubit-g** [comment]
@scottsand-db i've just accepted change suggested by @allisonport-db, what generated another commit. 
Will that do it?

**Comment by scottsand-db** [comment]
@pkubit-g  up that worked thanks

**Pull request merged**